chainer/variable.py

```diff
@@ -529,3 +529,22 @@ def __hash__(self):
         return super(Variable, self).__hash__()
 
     __array_priority__ = 200
+
+
+def get_array(arg):
+    """Returns ndarray data.
+
+    This method is helper function for :class:`~chainer.Variable`.
+
+    Args:
+        arg (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            If ``arg`` type is :class:`chainer.Variable`, it returns
+            ``arg.data``. Otherwise it returns ``arg``.
```

Comment:
This is a description of the returned value, and so it should be placed in the "Returns" section. Write the explanation of the argument itself here, like `"Object to extract an array from."`.